specification/deviceprovisioningservices/resource-manager/readme.java.md

```diff
@@ -0,0 +1,34 @@
+# Java DPS
+
+> see <https://aka.ms/autorest>
+
+This is the AutoRest configuration file for the Device Provisioning Service.
+
+## Java common settings
+
+These settings apply only when `--csharp` is specified on the command line.
```

Comment:
isn't this supposed to be java?

Reply:
copy pasta error. thanks for catching it.